docs/pages/2019_Xiamen/sections/first_taste.rst

```diff
@@ -0,0 +1,180 @@
+Then submit the calculation using:
+
+.. code:: bash
+
+    verdi run demo_calcjob.py
+
+From this point onwards, the AiiDA daemon will take care of your calculation.
+
+Analyzing the outputs of a calculation
+--------------------------------------
+
+.. code:: bash
+
+   verdi process list
+   verdi calcjob inputcat # (compare with python file)
+   verdi process list -a
+   verdi calcjob outputcat
+   verdi calcjob res
+
+Provenance
+
+.. code:: bash
+
+   verdi node show
+
+.. + put  pdf in web site + point out that this makes it reproducible)
+
+Moving to a different computer
+------------------------------
+
+Now, this Quantum ESPRESSO calculation ran on your (virtual) machine, which was already pre-configured in AiiDA.
+This works fine for test calculations but production runs you'll need to run on a compute cluster.
+
+For the purposes of this tutorial, you'll run on your neighbor's computer.
+ask IP address of your neighbor
+
+.. Add template to download (they just need to replace IP address
+
+
+.. literalinclude:: include/configuration/neighbor.yml
+
+
+.. note::
+
+    If you're completing this tutorial at a later time and have no partner machine,
+    simply use "localhost" instead.
+
+.. literalinclude:: include/configuration/neighbor-config.yml
+
+.. code:: bash
+
+  verdi computer setup --config neighbor.yml
+  verdi computer configure ssh neighbor --config neighbor-config.yml --non-interactive
+  verdi computer test
+
+.. Add template for code
+.. literalinclude:: include/configuration/qe.yml
+
+.. code:: bash
+
+  verdi code setup
+  verdi code list
+
+Now, modify the code label in your ``demo_calcjob.py`` script to use your newly set up code
+and run it again.
+
+.. code:: bash
+
+  verdi process list
+
+
+To see what is going on, AiiDA provides a command that lets you jump to the folder of the 
+calculation on the remote computer:
+
+.. code:: bash
+
+  verdi calcjob gotocomputer
+
+Have a look at the submission script ``...sh``.
+Notice the SLURM directives.
+
+From calculations to workflows
+------------------------------
+
+AiiDA can help you run individual calculations but AiiDA is really designed to
+help you run workflows that involve several calculations, while automatically
+keeping track of the provenance for full reproducibility.
+
+As the final step, we're going to launch the ``PwBandStructure`` workflow of the ``aiida-quantumespresso`` plugin.
+
+.. Add another python snippet
+
+.. code:: bash
+
+  verdi run demo_bands.py
+  verdi process list
+
+This workflow will:
+
+  #. Determine the primitive cell of the input structure
+  #. Run a calculation on the primitive cell to relax both the cell and the atomic positions (``vc-relax``)
+  #. Refine the symmetry of the relaxed structure to ensure the primitive cell is used 
+  #. Run a self-consistent field calculation on the refined structure
+  #. Determine a standard path of high symmetry k-points using `SeeK-path <https://www.materialscloud.org/work/tools/seekpath>`_
```

Comment:
seek path should come one line above, before the final SCF, because what is also does is to standardise the cell (so get the primitive, possibly rotate and change axes, ...). We also keep the standard path in memory to be used later (last step).

Reply:
I've made some changes, let me know.

Reply:
I think like this is ok, thanks